pkg/rtc/participant.go

```diff
@@ -143,6 +143,9 @@ type ParticipantParams struct {
 	PlayoutDelay                 *livekit.PlayoutDelay
 	SyncStreams                  bool
 	ForwardStats                 *sfu.ForwardStats
+
+	// Custom
+	ParticipantAddress string
```

Comment:
ParticipantAddress cannot be used this way. It may change over time as the user might switch their network (i.e. from WiFi to cellular) which results in IP change. You should somehow get the IP everytime from Transportmanager. Keep in mind that we have 2 PeerConnection per Participant so you might need to record two IPs since one PC can connect through TURN and the other directly.

Reply:
There is a ClientInfo field in the participant struct, and that is the only way I have found to get the client's address. Besides if the client changes IP it is considered a reconnection and thus expected to have the client info updated.